src/main/kotlin/graphics/scenery/controls/eyetracking/PupilEyeTracker.kt

```diff
@@ -192,7 +195,6 @@ class PupilEyeTracker(val calibrationType: CalibrationType, val host: String = "
                         if(poller.isReadable(socket)) {
                             val msg = ZMsg.recvMsg(socket)
                             val msgType = msg.popString()
-//                            logger.info("Received message type $msgType")
 
                             when(msgType) {
```

Comment:
Please let IntelliJ reformat this block, there's a bunch of spaces missing, impeding readability